spec/system/dashboard/contacts_spec.rb

```diff
@@ -0,0 +1,104 @@
+require 'rails_helper'
+
+RSpec.describe 'User management page', type: :system do
+  let(:user) { create(:user) }
+
+  before do
+    sign_in(user)
+  end
+
+  describe '#index' do
+    it 'should list all contacts of current account' do
+      account_contact = create(:contact, account: user.account)
+      other_contact = create(:contact)
+
+      visit 'dashboard/contacts'
+
+      expect(page).to have_text(account_contact.msisdn)
+      expect(page).not_to have_text(other_contact.msisdn)
+    end
+
+    it 'on click new contact will open new contact page' do
+      visit 'dashboard/contacts'
+
+      click_button 'New contact'
+      expect(page).to have_current_path(new_dashboard_contact_path)
+    end
+  end
+
+  describe '#new' do
```

Comment:
Try to name your spec and context from user perspective in system spec.
For example, here instead of writing "#new" write something like "user can create a contact"